pkg/client/client_test.go

```diff
@@ -594,6 +594,45 @@ func testReadOnlyMode(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+
+func testInetFunctionsFullAndSimplifiedQuery(t *testing.T) {
+
+	url := fmt.Sprintf("postgres://%s@%s:%s/%s?sslmode=disable", serverUser, serverHost, serverPort, serverDatabase)
+	client, _ := NewFromUrl(url, nil)
+
+
+	// Normally, client.Info() result should include columns for the values returned by inet_* function calls
+
+	info, _ := client.Info()
+	assert.Contains(t, info.Columns, "session_user")
+	assert.Contains(t, info.Columns, "inet_client_addr")
+	assert.Contains(t, info.Columns, "inet_client_port")
+	assert.Contains(t, info.Columns, "inet_server_addr")
+	assert.Contains(t, info.Columns, "inet_server_port")
+
+	// Prepare our restricted database and revoke execution permissions on inet_client_addr() function
+	client.Query("\nCREATE USER IF NOT EXISTS testuser WITH PASSWORD 'secret';\n")
```

Comment:
use `client.db.MustExec(...`) for setup calls since we want them to break the test suite if they're not successfully executed